vlib/x/templating/dtm/README.md

```diff
@@ -0,0 +1,193 @@
+# dtm - Dynamic Template Manager
+
+A simple template manager integrated into the V project, designed to combine the power of V
+templates with Vweb, without the need to recompile the application with every change.
+
+**_This module is in the experimental phase, it lacks some features and may contain bugs_**
+
+## Quick Start
+
+Using the dynamic template manager ( named '**dtm**' in this readme) is relatively straightforward.
+You just need to create an instance and initialize it. Then, call the '**serve_dynamic_template**'
+function in the code that manages your web pages. 
+
+Before starting, at the root directory of your vweb project, you need to create a '**templates**'
+folder as well as a '**vcache**' folder. The absence of these directories at the root of your
+project will prevent you from using the dtm.
+
+Next, you must add your HTML file into the folder you previously created. ***Be aware that if the
+HTML templates are not placed in the correct directory, the DTM will return an error message
+indicating that it cannot find the template!!***
+
+A minimal quick start example:
+
+```
+import vweb
+import x.templating.dtm
+
+struct App {
+	vweb.Context
+pub mut:
+    // The vweb_global attribute must be applied in order to use the dtm 
+    // (Dynamic Template Manager) throughout your application.
+    // However, you do not need the attribute if you are using x.vweb
+	dtm &dtm.DynamicTemplateManager = dtm.create_dtm() @[vweb_global] 
+}
+
+fn main() {
+	mut app := &App{}
+
+	dtm.initialize_dtm(mut app.dtm)!
+
+	vweb.run(app, 18081)
+}
+
+@['/']
+pub fn (mut app App) index() vweb.Result {
+	html_content := app.dtm.serve_dynamic_template('index.html')
+	return app.html(html_content)
+}
+
+```
+
+You have a ready-to-view demonstration available
+[here](https://github.com/vlang/v/tree/master/vlib/vweb/tests/dynamic_template_manager_test_server).
+
+## Available Options
+
+There are two types of option possibilities:
+
+- Specific to the initialization
+- Defined for each web page
+
+### Specific to the initialization
+
+Three parameters are available:
+
+- `max_size_data_in_mem` : ( **Int** value ) Maximum size of data allowed in memory for each cached
+  template. The value must be specified in kilobytes. ( Default is: 500KB / Limit max is : 500KB )
+- `compress_html` : ( **Bool** value ) Light compress of the HTML ouput, to remove all unnecessary
+  spacing. ( Default is true )
+- `active_cache_server` : ( **Bool** value ) Activate or not the template cache system. ( Default is
+  true, ***_Highly recommended to keep it enabled for optimal performance_*** )
+
+Using like this :
+
+```
```

Comment:
use this here:
```
```v ignore
```

Reply:
It will still get highlighted, but since it is a snippet, it will not get compiled, and `v check-md .` will not complain about it.